packages/ember-htmlbars/tests/helpers/get_test.js

```diff
@@ -1,495 +0,0 @@
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '[red and yellow] [red and yellow]', 'should return \'red and yellow\' for {{get colors "apple.gala"}}');
-
-  run(function() {
-    view.set('context.colors', { apple: { gala: 'yellow and red striped' }, banana: 'purple' });
-  });
-
-  equal(view.$().text(), '[yellow and red striped] [yellow and red striped]', 'should return \'yellow and red striped\' for {{get colors \'apple.gala\'}}');
-
-  run(function() {
-    view.set('context.colors.apple.gala', 'yellow-redish');
-  });
-
-  equal(view.$().text(), '[yellow-redish] [yellow-redish]', 'should return \'yellow-redish\' for {{get colors \'apple.gala\'}}');
-});
-
-QUnit.test('should be able to get an object value with a bound/dynamic key', function() {
-  var context = {
-    colors: { apple: 'red', banana: 'yellow' },
-    key: 'apple'
-  };
-
-  view = EmberView.create({
-    context: context,
-    template: compile('[{{get colors key}}] [{{if true (get colors key)}}]')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '[red] [red]', 'should return \'red\' for {{get colors key}}  (key = \'apple\')');
-
-  run(function() {
-    view.set('context.key', 'banana');
-  });
-
-  equal(view.$().text(), '[yellow] [yellow]', 'should return \'red\' for {{get colors key}} (key = \'banana\')');
-
-  run(function() {
-    view.set('context.colors', { apple: 'green', banana: 'purple' });
-  });
-
-  equal(view.$().text(), '[purple] [purple]', 'should return \'purple\' for {{get colors key}} (key = \'banana\')');
-
-  run(function() {
-    view.set('context.key', 'apple');
-  });
-
-  equal(view.$().text(), '[green] [green]', 'should return \'green\' for {{get colors key}} (key = \'apple\')');
-
-  run(function() {
-    view.set('context.colors.apple', 'red');
-  });
-
-  equal(view.$().text(), '[red] [red]', 'should return \'red\' for {{get colors key}}  (key = \'apple\')');
-});
-
-QUnit.test('should be able to get an object value with nested dynamic key', function() {
-  var context = {
-    colors: { apple: { gala: 'red and yellow', mcintosh: 'red' }, banana: 'yellow' },
-    key: 'apple.gala'
-  };
-
-  view = EmberView.create({
-    context: context,
-    template: compile('[{{get colors key}}] [{{if true (get colors key)}}]')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '[red and yellow] [red and yellow]', 'should return \'red and yellow\' for {{get colors "apple.gala"}}');
-
-  run(function() {
-    view.set('context.key', 'apple.mcintosh');
-  });
-
-  equal(view.$().text(), '[red] [red]', 'should return \'red\' for {{get colors \'apple.mcintosh\'}}');
-
-  run(function() {
-    view.set('context.key', 'banana');
-  });
-
-  equal(view.$().text(), '[yellow] [yellow]', 'should return \'yellow\' for {{get colors \'banana\'}}');
-});
-
-QUnit.test('should be able to get an object value with subexpression returning nested key', function() {
```

Comment:
:heavy_check_mark: should be able to get an object value with subexpression returning nested key